pipeline/outputs/dynatrace.md

```diff
@@ -0,0 +1,61 @@
+| format                     | The data format to be used in the HTTP request body                                                                                                                                                                                 | json                                               |
+| json_date_format           | Date format standard for JSON                                                                                                                                                                                                       | iso8601                                            |
+| json_date_key              | Fieldname specifying message timestamp                                                                                                                                                                                              | timestamp                                          |
+| tls                        | Specify to use TLS                                                                                                                                                                                                                  | on                                                 |
+| tls.verify                 | TLS verification                                                                                                                                                                                                                    | on                                                 |
+
+## Getting Started
+
+1. Get a [Dynatrace API](https://docs.dynatrace.com/docs/shortlink/api-authentication) token with the `logs.ingest` (Ingest Logs) scope.
+2. Determine your Dynatrace [environment ID](https://docs.dynatrace.com/docs/shortlink/monitoring-environment#environment-id).
+3. In your main Fluent Bit configuration file, append the following _Output_ section:
+
+```text
+[OUTPUT]
+    name         http
+    match        *
+    header       Content-Type application/json; charset=utf-8
+    header       Authorization Api-Token {your-API-token-here}
+    allow_duplicated_headers false
+    host         {your-environment-id}.live.dynatrace.com
+    Port         443
+    URI          /api/v2/logs/ingest
+    Format       json
+    json_date_format iso8601
+    json_date_key timestamp
+    tls          On
+    tls.verify   On
+```
+
+4. Once the configuration is applied and new logs are appended, verify their receipt within Dynatrace. This can be done using Dynatrace [Notebooks](https://docs.dynatrace.com/docs/observe-and-explore/dashboards-and-notebooks/notebooks) with a Dynatrace Query Language query, for example:
+
+    ```
+    fetch logs
+    ```
```

Comment:
This set of changes puts in line wraps (to help with formatting the raw content), renumbers the lines (Markdown supports using all 1s, which will autonumber lists), and implements a few more minor changes.

```suggestion
## Getting started

To get started with sending logs to Dynatrace:

1. Get a [Dynatrace API](https://docs.dynatrace.com/docs/shortlink/api-authentication)
   token with the `logs.ingest` (Ingest Logs) scope.
1. Determine your Dynatrace
   [environment ID](https://docs.dynatrace.com/docs/shortlink/monitoring-environment#environment-id).
1. In your main Fluent Bit configuration file, append the following `Output` section:

   ```text
   [OUTPUT]
       name         http
       match        *
       header       Content-Type application/json; charset=utf-8
       header       Authorization Api-Token {your-API-token-here}
       allow_duplicated_headers false
       host         {your-environment-id}.live.dynatrace.com
       Port         443
       URI          /api/v2/logs/ingest
       Format       json
       json_date_format iso8601
       json_date_key timestamp
       tls          On
       tls.verify   On
   ```

1. After the configuration is applied and new logs are appended, verify their receipt
   within Dynatrace. This can be done using Dynatrace
   [Notebooks](https://docs.dynatrace.com/docs/observe-and-explore/dashboards-and-notebooks/notebooks)
   with a Dynatrace Query Language query. For example:

   ```bash
   fetch logs
   ```
```